scripts/npmDepsHash.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const childProcess = require('child_process');
+
+/* eslint-disable no-console */
+async function main(_argv = process.argv) {
+  const projectRoot = path.join(__dirname, '..');
+
+  childProcess.exec(
+    'nix-shell -p prefetch-npm-deps --run "prefetch-npm-deps ./package-lock.json" | tail -n 1',
+    async (error, stdout, _) => {
+      if (error) {
+        console.error(`error: ${error.message}`);
+        return;
+      }
+
+      const hash = stdout.trim();
```

Comment:
Why is this needed? Just to eliminate the NL at the end? Generally NL is ok, but readers should trim not writers.

Reply:
NLs are legitimate file endings. Readers should trim. Not writers to a file.

Reply:
I just tested without it and things seem to work fine. I've removed the trim.